test/bddtests/fixtures/config/didDocumentTemplate.json

```diff
@@ -0,0 +1,26 @@
+
+{
+  "@context": "https://w3id.org/did/v1",
+  #DIDID
```

Comment:
we should have proper document without DIDID as template, DID ID can be added dynamically for negative tests

Reply:
done